Formula/xcftools.rb

```diff
@@ -0,0 +1,52 @@
+class Xcftools < Formula
+  desc "Tools for GIMP's native file format XCF"
+  homepage "http://henning.makholm.net/software"
+  url "http://henning.makholm.net/xcftools/xcftools-1.0.7.tar.gz"
+  sha256 "1ebf6d8405348600bc551712d9e4f7c33cc83e416804709f68d0700afde920a6"
+
+  depends_on "libpng"
+
+  def install
+    # Avoid deprecated use of libpng 1.2
+    inreplace "xcf2png.c", /png_(voidp|error_ptr)_NULL/, "NULL"
```

Comment:
I just sent an email to the author about these issues. Not sure if he is going to get back anytime soon, but hope to see this formula in Homebrew given that it's still well alive in Debian and FreshPorts.